Tasks/SqlAzureDacpacDeploymentV1/Utility.ps1

```diff
@@ -116,6 +106,7 @@ function Get-SqlPackageCommandArguments
         SourcePassword = "/SourcePassword:";
         TargetFile = "/TargetFile:";
         OutputPath = "/OutputPath:";
+        AccessToken = "/AccessToken:";
```

Comment:
Set the value of $token to be **** when isOutputSecure flag is set, otherwise it will leak secrets into the logs.